tests/python_tests/test_vlm_pipeline.py

```diff
@@ -403,6 +415,14 @@ def test_images(request: pytest.FixtureRequest):
     return [request.getfixturevalue(image) for image in request.param]
 
 
+@pytest.fixture(scope="function", params=[
+    pytest.param([], id="no_videos"),
+    pytest.param(["synthetic_video_32x32_tensor"], id="single_video"),
+    pytest.param(["synthetic_video_32x32_tensor", "video_frame_3_tensor"], id="multiple_videos"),
+])
+def test_videos(request: pytest.FixtureRequest):
```

Comment:
```suggestion
def synthetic_video(request: pytest.FixtureRequest):
```
Remove `test` from the name